assets/tests/QuestionSchemaValidationServiceSpec.js

```diff
@@ -47,6 +47,18 @@ describe('QuestionSchemaValidationService', function() {
     });
   }));
 
+  describe('validateTitlesAreUnique', function() {
+    it('should verify that all questions have a unique title', function() {
+      var titles = new Set();
+      expect(questions.every(function(question) {
+        var title = question.getTitle();
+        var isUnique = !(title in titles);
+        titles.add(title);
+        return isUnique;
+      })).toBe(true);
```

Comment:
I mildly think it's simpler/cleaner to dump all titles into a set and verify the set size matches the list length, but happy to defer to you on this.